src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsauditapi/resource/AuditResourceTest.kt

```diff
@@ -162,7 +179,7 @@ class AuditResourceTest : IntegrationTest() {
         .expectStatus().isAccepted
 
       verify(auditQueueService).sendAuditEvent(auditEvent)
-//      verify(awsSqsClient).sendMessage(eq(auditQueueConfig.queueUrl), anyString())
+      // test call to queue
```

Comment:
this current verify method no longer works as the aws sendMessage now includes sendMessage builder and more parameters.
These will be updated and re added in next PR